sections/errorhandling/useonlythebuiltinerror.md

```diff
@@ -2,7 +2,7 @@
 
 ### One Paragraph Explainer
 
-The permissive nature of JS along with its variety code-flow options (e.g. EventEmitter, Callbacks, Promises, etc) pushes to great variance in how developers raise errors – some use strings, other define their own custom types. Using Node.js built-in Error object helps to keep uniformity within your code and with 3rd party libraries, it also preserves significant information like the StackTrace. When raising the exception, it’s usually a good practice to fill it with additional contextual properties like the error name and the associated HTTP error code. To achieve this uniformity and practices, consider extending the Error object with additional properties, see code example below
+The permissive nature of JS along with its variety of code-flow options (e.g. EventEmitter, Callbacks, Promises, etc) pushes to great variance in how developers raise errors – some use strings, other define their own custom types. Using Node.js built-in Error object helps to keep uniformity within your code and with 3rd party libraries, it also preserves significant information like the StackTrace. When raising the exception, it’s usually a good practice to fill it with additional contextual properties like the error name and the associated HTTP error code. To achieve this uniformity and practices, consider extending the Error object with additional properties, see code example below
```

Comment:
Maybe we can remove the `JS` abbreviation now and use `JavaScript`, which makes it more readable as well 👍

Reply:
I agree, and will make the change.  Thanks @BrunoScheufler.

Reply:
@BrunoScheufler, #422 is the pull request that addresses your recommended change.